src/libs/fsdrv/lv_fs_win32.c

```diff
@@ -17,17 +17,9 @@
 /*********************
  *      DEFINES
  *********************/
-#if LV_FS_WIN32_LETTER == '\0'
-    #error "LV_FS_WIN32_LETTER must be set to a valid value"
-#else
-    #if (LV_FS_WIN32_LETTER < 'A') || (LV_FS_WIN32_LETTER > 'Z')
-        #if LV_FS_DEFAULT_DRIVE_LETTER != '\0' /*When using default drive letter, strict format (X:) is mandatory*/
-            #error "LV_FS_WIN32_LETTER must be an upper case ASCII letter"
-        #else /*Lean rules for backward compatibility*/
-            #warning LV_FS_WIN32_LETTER should be an upper case ASCII letter. \
-            Using a slash symbol as drive letter should be replaced with LV_FS_DEFAULT_DRIVE_LETTER mechanism
-        #endif
-    #endif
+
+#if !LV_FS_IS_VALID_LETTER(LV_FS_STDIO_LETTER)
```

Comment:
```suggestion
#if !LV_FS_IS_VALID_LETTER(LV_FS_WIN32_LETTER)
```

Reply:
Nice catch. I double checked and still missed this one. :(